scene/2d/tile_map.h

```diff
@@ -382,15 +462,20 @@ class TileMap : public Node2D {
 	virtual Rect2 _edit_get_rect() const override;
 #endif
 
+#ifndef DISABLE_DEPRECATED
+	// Deprecated methods
```

Comment:
```suggestion
	// Deprecated methods.
```
The comment is unnecessary tbh.